cpp/models/ide_secir/simulation.h

```diff
@@ -126,19 +115,17 @@ class Simulation
 
 private:
     std::unique_ptr<Model> m_model; ///< Unique pointer to the Model simulated.
-    ScalarType m_t0; ///< Start time used for simulation.
     ScalarType m_dt; ///< Time step used for numerical computations in simulation.
 };
 
 /**
  * @brief simulates a compartmental model
- * @param[in] t0 start time
  * @param[in] tmax end time
  * @param[in] dt initial step size of integration
  * @param[in] model an instance of a compartmental model
  * @return a TimeSeries to represent the final simulation result
  */
```

Comment:
I made a small update to this description in another branch, for consistency you could use
```cpp
/**
 * @brief Run a Simulation of a mio::isecir::Model.
 * To set the start time t0, use the model parameter StartDay.
 * @param[in] tmax End time.
 * @param[in] dt Initial step size of integration.
 * @param[in] model An instance of mio::isecir::Model.
 * @return A TimeSeries to represent the final simulation result
 */
```